project-templates/ecs/basic/README.md

```diff
@@ -0,0 +1,161 @@
+# Waypoint Project Template Example
+
+This is an example of how to create and use project templating in HCP Waypoint.
+
+Pre-Requisites
+
+AWS Account with Admin Permissions
+
+Terraform Enterprise Account with No Code Module Enabled.
+
+## Initial Setup
+
+### Terraform Module Creation
+
```

Comment:
I think we need a blurb here explaining what the point of this module is.

Idea:


```suggestion
Now that we've created the base shared infrastructure, we can move on to app-specific infrastructure.

Most applications need some long-lived infrastructure, like a container registry and a load balancer. This module gives platform engineers a space to define this app-specific infrastructure for their specific organization.

This example extends Waypoint's [ecs module](https://registry.terraform.io/modules/hashicorp/waypoint-ecs/aws/latest) to create a single global container registry, and resources like an ALB, security group, and IAM role in the dev and prod environments.

This module is an opportunity for platform engineers to get creative and define any other app-specific infrastructure, like  monitoring dashboards or tickets in a ticketing system. 
```

When the advanced example lands, we should link to it here.